index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/mobile.css">
+    <link rel="stylesheet" href="css/desktop.css">
+    <link rel="shortcut icon" href="images/classic-logo.png" type="image/x-icon">
+    <title>First Capstone Project</title>
+</head>
+<body>
+    <!-- Navbar -->
+    <a href="#" id="hamburguer-button" class="desktop-hide"><i class="fa-sharp fa-solid fa-bars"></i></a>
+    <nav id="navbar" class="navbar">
+        <div class="social-media-bar mobile-hide">
+            <a href="https://www.facebook.com/mrolympiallc/?locale=es_LA"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="https://twitter.com/MrOlympiaLLC"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="https://github.com/NicolasAndrehh">My Page</a>
+            <a href="#">Logout</a>
+        </div>
+        <div class="menu-bar">
+            <a href="#" id="close-menu" class="desktop-hide"><i class="fa-solid fa-xmark"></i></a>
+            <a href="index.html"><img src="images/mrOlympiaLogo.png" alt="" class="mobile-hide"></a>
+            <ul class="menu-links">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#categories-section">Categories</a></li>
+                <li><a href="#athletes-section">Athletes</a></li>
+                <li><a href="#sponsors-section">Sponsors</a></li>
+                <li><a href="about.html" class="desktop-hide">Learn more</a></li>
+            </ul>
```

Comment:
[OPTIONAL]

- Your navigation items may not match the arrangement on the design guidelines as yours is missing the About navigation item. It will be nice if you are consistent with the design guideline 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/217648797-05ad4034-294a-46dd-8164-6fbb052fb244.png)

Design guildeline👇🏽 
![image](https://user-images.githubusercontent.com/63473584/217648997-4fe9bc89-d75e-46dc-9593-c2720d38bec1.png)

Reply:
You're right I'll do the change, thanks for letting me know! 😄